docs/tremor-query/index.md

```diff
@@ -120,6 +120,13 @@ Tremor supports tumbling windows by number of events or by time.
 General configuration Parameters:
 
 * `eviction_period`: duration in nanoseconds without events arriving, after which to evict / remove the current window data for a single group.
+* `max_groups`: maximum number of groups to maintain simultaneously in memory. Groups added beyond that number will be ignored. Per default, tremor allows as much groups as will fit into memory.
```

Comment:
I think this needs re-wording, by default tremor does not limit the number of groups if a user runs out of memory it swaps and then eventually crashes with an out of memory error